### PR TITLE
setting up eks logging for fargate and fluent bit

### LINKED ISSUE #444: EKs Set up fluentBit and fargateLogging
To be able to setup proper logging for eks, we currently have 2 functions in deploylibs `k8s/add_ons` directory
- enableFargateLogging
- installFluentBitForCW
We need these to be replicated to the new format in the kubernetes or aws plugin.

Definition of done.
- The cluster has both of these pieces of functionality working.
- Manifests are installed and output to the compiled dir

### PR DESCRIPTION
**Pull request by jhsinger-klotho**
• Does any part of it require special attention?
• Does it relate to or fix any issue? closes #444 

sets up fargate logging and fluent bit for container insights.

We now can install manifests via ConfigFile in pulumi.

We also output our manifests to the `compiled/manifests` directory

### Standard checks

- **Unit tests**: Any special considerations? added test case
- **Docs**: Do we need to update any docs, internal or public?
- **Backwards compatibility**: Will this break existing apps? If so, what would be the extra work required to keep them working?

### CODE REVIEW

File: pkg/cli/klothomain.go
Comment (yuval-klotho) [inline pkg/cli/klothomain.go:257]: 😬 lol

File: pkg/provider/aws/resources/eks.go
Comment (yuval-klotho) [inline pkg/provider/aws/resources/eks.go:100]: instead of (or in addition to, your pick) just printing the err, also add it as a zap field (`zap.Error(err)` or something like that, I forget exactly). That will let us get the full stack trace on telemetry.

ditto below

**Review by yuval-klotho — COMMENTED** [review]
lgtm after the logging thing. no need for second review/approval from me after that, though.

File: pkg/cli/klothomain.go
Comment (jhsinger-klotho) [reply pkg/cli/klothomain.go:257]: yeah i saw that as i was debugging... not sure when that got commented out

File: pkg/provider/aws/resources/eks.go
Comment (jhsinger-klotho) [reply pkg/provider/aws/resources/eks.go:100]: well wouldnt we actually fail compilation with zap.Error? The reason i opted for warn is because these charts just affect streaming logs to CW and wouldnt affect any bit of the functionality

**Review by yuval-klotho — APPROVED** [review]
hitting the approve button so that GH won't yell at you when you try to merge, since my last review was really a soft-approve anyway

File: pkg/provider/aws/resources/eks.go
Comment (yuval-klotho) [reply pkg/provider/aws/resources/eks.go:100]: I'm not saying log at error level, I'm just saying attach the field. When we send the telemetry, we look at the fields, and any that are of type `error` we extract the full stack trace from and send that.

**Pull request merged**
